.github/workflows/patch.yml

```
@@ -0,0 +1,49 @@
+name: Patch
+on:
```

Easy way to re-run if we want to for whatever reason

```suggestion
on:
	workflow_dispatch:
```